docs/sections.md

```diff
@@ -20,7 +20,7 @@ Sections help to maintain vertical rhythm.
 </div>
 
 ```html
-<div class="section">
+<div class="section section--first">
```

Comment:
Not sure I follow, since using something like `push35--top` is a single class that has a fixed value for spacing.